gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.openstreetmap.atlas
-version=4.0.8-SNAPSHOT
+version=4.0.9-SNAPSHOT
```

Comment:
I thought this was updated after release is done?

Reply:
4.0.8 is out
was missing newest generator tho.